test/unittest_images.cxx

```diff
@@ -295,4 +295,4 @@ Fl_RGB_Image *ImageTest::i_ga = 0;
 Fl_RGB_Image *ImageTest::i_rgb = 0;
 Fl_RGB_Image *ImageTest::i_rgba = 0;
 
-UnitTest images("drawing images", ImageTest::create);
+UnitTest images(7, "drawing images", ImageTest::create);
```

Comment:
If I understand (guess) this correctly "7" is the index in the list of unittests (left box). I suggest to use named constants (enum) for the index. This enum could be defined in a central place (header file) so index changes would be easier (inserting a new test at an arbitrary place).

Reply:
Makes sense. I'll do that.

Reply:
Done in my repo